ocaml/xapi/xapi.ml

```diff
@@ -1138,7 +1150,16 @@ let server_init () =
               (* Try to say hello to the pool *)
               match attempt_pool_hello ip with
               | None ->
-                  finished := true
+                  if compare_xapi_version_with_coordinator ~__context > 0 then (
+                    error
```

Comment:
I think an alert would be much nicer for users.

Reply:
To add an alert for this kind of pool member xapi startup failure, I think we also need to remove the alert later after the pool coordinator xapi got updated and restarted which un-blocks the pool member xapi startup.
Will add and remove alert in this way.